dev/drivers/scripts/analyses/plots/jevs_analyses_grid2obs_plots.sh

```diff
@@ -28,7 +28,8 @@ module load prod_envir/${prod_envir_ver}
 
 export envir=prod
 export DATAROOT=/lfs/h2/emc/stmp/${USER}/evs_test/$envir/tmp
-export KEEPDATA=NO
+export KEEPDATA=YES
+export SENDMAIL= YES
```

Comment:
@PerryShafran-NOAA Super tiny, but can this extra space be deleted?

Reply:
That's good lookin' out!  I removed that space.